further_link/endpoint/run.py

```diff
@@ -150,6 +195,63 @@ async def on_display_activity(connection_details: VncConnectionDetails):
         self.process_handlers[process_id] = handler
 
 
+bt_run_manager: dict = {}
+
+
+async def bluetooth_run_handler(device, uuid, message, characteristic_to_report_on):
+    try:
+        message_dict = bytearray_to_dict(message)
+    except Exception:
+        msg = "Error: invalid format"
+        logging.error(msg)
+        await device.write_value(msg, characteristic_to_report_on)
+        return
+
+    client_uuid = message_dict.pop("client_uuid", None)
+
+    if client_uuid is None:
+        msg = "Error: client_uuid not provided in message"
+        logging.error(msg)
+        await device.write_value(msg, characteristic_to_report_on)
+        return
+
+    global bt_run_manager
+    if bt_run_manager.get(client_uuid) is None:
+
+        async def send_func(message):
+            logging.debug(f"Sending: {message[0:120]}")
+            await device.write_value(message, characteristic_to_report_on)
+
+        bt_run_manager[client_uuid] = RunManager(send_func, user=None, pty=True)
```

Comment:
can we not pass the user and pty options somehow? forget where the conversation went on that. it's not essential right now since they're not actually configurable on the FE yet, but I think the user one may be useful in future